src/client.rs

```diff
@@ -52,33 +146,31 @@ pub fn get_client(config: &crate::config::Config) -> Result<oxide::Client> {
         None
     };
 
-    // Attempt to read `hosts.toml` and extract a token from it. If this fails
-    // for any reason (`hosts.toml` not found or malformed, or no search path
+    // Attempt to read credentials config and extract a token from it. If this fails
+    // for any reason (`credentials/hosts.toml` not found or malformed, or no search path
```

Comment:
`credentials/hosts.toml` reads like a path to me, but I think the intention is `credentials.toml or hosts.toml`?

Reply:
That's right, fixed.